extensions/aerial_photo/README.md

```diff
@@ -0,0 +1,20 @@
+# Aerial Photography Extension Specification
+
+**This is a work in progress and has not been reviewed everything is likely to change**
+
+- **Title**: aerial_photo
+- **Identifier**:
+  [https://linz.github.io/stac/_STAC_VERSION_/film/aerial_photo.json]()
+- **Field Name Prefix**: aerial_photo
```

Comment:
The prefix should use a hyphen instead of an underscore to align with existing STAC extensions.

Reply:
e.g. https://github.com/stac-extensions/ml-aoi

Reply:
Note: I realise I led you astray with `historical_imagery`.